pkg/operator/operators/klusterlet/options.go

```diff
@@ -52,6 +56,24 @@ func (o *Options) RunKlusterletOperator(ctx context.Context, controllerContext *
 
 	kubeInformer := informers.NewSharedInformerFactory(kubeClient, 5*time.Minute)
 
+	// to reduce cache size if there are larges number of secrets
+	newOnTermInformer := func(name string) informers.SharedInformerFactory {
+		return informers.NewSharedInformerFactoryWithOptions(kubeClient, 5*time.Minute,
+			informers.WithTweakListOptions(func(options *metav1.ListOptions) {
+				options.FieldSelector = fields.OneTermEqualSelector("metadata.name", name).String()
```

Comment:
Is there any expression of the selector to make it possible to be logic OR for the `metadata.name`?

Reply:
have not found that expression.